### PR TITLE
add solana-passkeys example with fee payer

### PR DESCRIPTION
**Pull request by andrewkmin**
## Summary & Motivation
$title

## How I Tested These Changes

https://github.com/user-attachments/assets/8a79aab1-3d4f-45d4-8e48-2ccb8811b77e

## Did you add a changeset?

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

File: examples/with-solana-passkeys/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "eslint": "8.43.0",
+    "eslint-config-next": "14.2.10",
+    "esm": "^3.2.25",
+    "install": "^0.13.0",
+    "next": "14.2.10",
+    "npm": "^9.7.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-hook-form": "^7.45.1",
+    "tweetnacl": "^1.0.3",
```
Comment (moe-dev) [inline examples/with-solana-passkeys/package.json:35]: Nit: we don't use this atm - although should we? Maybe referencing the website makes more sense

File: examples/with-solana-passkeys/src/pages/api/createSubOrg.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { Turnkey, TurnkeyApiTypes } from "@turnkey/sdk-server";
+import { refineNonNull } from "@/utils";
+import { TWalletDetails } from "@/types";
+
+// Default path for the first Ethereum address in a new HD wallet.
```
Comment (moe-dev) [inline examples/with-solana-passkeys/src/pages/api/createSubOrg.ts:6]: NIT: Solana addresses, Coin type 501

File: examples/with-solana-passkeys/src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+            <input
+              className={styles.button}
+              type="submit"
+              value="Sign and Broadcast Transaction"
+            />
+          </form>
+          {signedTransaction && (
+            <div className={styles.info}>
+              <p>
+                🚀 Transaction broadcasted and confirmed!
```
Comment (moe-dev) [inline examples/with-solana-passkeys/src/pages/index.tsx:399]: 🚀 🚀 🚀 🚀 🚀

File: packages/sdk-server/src/index.ts [hunk 4]
```diff
@@ -64,13 +66,14 @@ export {
...
   defaultEthereumAccountAtIndex,
+  defaultSolanaAccountAtIndex,
   getWebAuthnAttestation,
   sealAndStampRequestBody,
   signWithApiKey,
 };
 
 // Constants
-export { DEFAULT_ETHEREUM_ACCOUNTS };
+export { DEFAULT_ETHEREUM_ACCOUNTS, DEFAULT_SOLANA_ACCOUNTS };
```
Comment (moe-dev) [inline packages/sdk-server/src/index.ts:76]: 🧹

**Review by moe-dev — APPROVED** [review]
Small nits. This is such a clutch example though 👑

File: examples/with-solana-passkeys/package.json (hunk 1 above)
Comment (andrewkmin) [reply examples/with-solana-passkeys/package.json]: removed!

**Pull request merged**